eventprocessorhost/eh_config.py

```diff
@@ -0,0 +1,49 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# -----------------------------------------------------------------------------------
+
+import time
+import urllib
+import hmac
+import hashlib
+import base64
+
+class EventHubConfig:
+    """
+    A container class for event hub properties. Takes sb_name (service bus name),
+    eh_name (event hub name), policy (URL-encoded-SAS-policy),
+    sas_key(URL-encoded-SAS-key), consumer_group
+    """
+    def __init__(self, sb_name, eh_name, policy, sas_key, consumer_group):
+        self.sb_name = sb_name
+        self.eh_name = eh_name
+        self.policy = policy
+        self.sas_key = sas_key
+        self.consumer_group = consumer_group
+        self.client_address = self.get_client_address()
+        self.rest_token = self.get_rest_token()
+        
+
+    def get_client_address(self):
+        """
+        Returns an auth token dictionary for making calls to eventhub
+        REST API.
+        """
+        return ("amqps://{}:{}:{}@{}.servicebus.windows.net/"\
```

Comment:
@aribornstein The way the address is formed does not look right to me. Could you double-check?

Reply:
yep fixed :)